### ISSUE TITLE
Comment in method chain gets deleted

### ISSUE THREAD

**Issue by pbiggar** [fsprojects/fantomas#2062]
Issue created from [fantomas-online](https://fsprojects.github.io/fantomas-tools/#/fantomas/preview?data=N4KABGBEDGD2AmBTSAuKAlWAbLAjAhgE4A6AdhAHSY4GEAys0%2BALrCeWFdnkQJKkBnZvlLREZCAHpJnAIICAQlkYBrAJakA5g02bEhABQBlAJ5DEAWwoAVNRcRGADiIoAxQrAtHEcUvAFgAKwAlBKcAKKEHobO0Cr4egA0YNAArkKewZCJ4FACiMzMGpoCqGAA2rkQoBC1UAAkzCaOyGiQGszZVbWQ9QBu%2BFiprRXddQAMOXV1kPxIpMxGagBeyFPTEAAsY2AAut0AvuvVO71NLWXtC10bvQNDI5UbEACMx9OQALL4AB50Gog6IgtMwABY3Z4vABM4x2%2BzqR26NQ%2BjWaI0gwPgAHkAGb-UgOJpYNanfqDYZlJ7PKHvGbhPy4-Ek55QLA4yBww7vZEzVEXNq4WDYCG8%2B4UtBUjYAZlpPX4%2BUIzFcGkGADlEAB3LAAkXTZiEYachHc0nndGC4Wyu7kx47LayiCQbwWNRwZSkWTMenYvE6h1gHGDfJG2qIuo8np881CrC6x1kh6Uu1Bf1O2KIBSIHFsRAABSI%2BHszH0cdq%2BsNG3hoZNtyjlwtsatCfFoxZADZU058GJM9nCIDYBr9Ex8vw%2BowWGpYKRSxBy4gQxAw7UI-GzfWY6XrYmJcmAOyd9O9nMAVUcLUII8QY4nRWns4DQYXla5SNNaI3ltJYttLIAHIe3YZlmOYAMJYPgAgCKB05CAa0CsIQD6BlgwYvsab61uuAqbk2P5JiyACcgE9iB-afJYuAlv6KFodMVZLjWKLYVADZbs2v6QpMyZpkBx79jB7rIU%2Bi5gMuJxYR%2BOFfrW%2BG7iyLxvDxXZiLIOLFoQMEWBY%2BAPvOoniWAq4NCxkBsXhNoEZCNLKUeZEOJYrrYPeNEieh1aYcxUmsbh36WfJkIyrZQFqRpzpOUJ-r6e5jGebypnmX5O6tpCmypnMwLMFipDWIQJgAOpqGCwmoc%2B9GvuG778j5MkonJKUbC8gQkYgsgeKkfgACKINqLoaXpBplXUDFiUx8XeVcnQWclkrTC8HY8d8Py8Di1igsC4SlUYoJsMwhXwMV-qbLCMWjXFkamR07H1bNdQvAei2-PwOJqD8WIXiwbDhD8jj9lBU4zv6gQneVGGVZJ1WTdd-kNXNAGPT86A%2BGw8D7YdybHQZY0XRNV3TS2t21C8xEI0jcCEPAqqpBYVGELivDFhYpT%2Bi8WPnWuE0WKkWBFNqBKuGwOmFPo1gfvjnEbDCqZkyjnzc7zAIC4QQv9am0CggWCH6AA%2BhqagHeCbPg15kN40lBPJlCSksl8vxtYQ%2BAmFi9BqEIaPgkdIPDRVK5VeiZuyTDhMQFCNk20t9uO87-xCFTNP6PTjPM8mrOnYZxlnJz8tqHziBKyrIti%2BbEvTFCQU25HTsu0Ics8zniuCywqs8ermsabr%2BvFUbvsQ-71zi1ZktpQjABqNoKBo8DFO7D5-l7tQjenfuXAHdVB5bzUI64HUIQDE9%2BNPncexj88QIv2Mc6b-fF4PpcLeHvydbAzAAOIFN9v2IP904z0Dp97D7CSJtoy1VFOvFkUIHrh2zrnJQqgFAOziAUAQ2UjCFkQIJamgNky0SGgvQBRll7SUbDfAKkt4Y23VFqAEmZmBDmBKLFo3UXqkCKgDWQfgKLx0IMnFkuDu5APGpDRKgcZqWxJjbAA0ogRAjgVprWBPwNB9gmQlTot7MGPdgEr2vqIi2LIpTcQfj8TaLoBCgl-ifARhDe6fhIXokudQpTW2eE6YoxI2qaGpplfKiBcCfAQMyZ4-C04XxMhNERa8xEGLDq42Q2pNCkG3qIO8pAliJJYKkfs1hYAZQWJOFyOC3Kgw8sbIRID7FRP0c8KU5c4k830KQScfQBxaC4bTZhGg2GwTUXgs%2BBCM51mIdDaJNTh7QLrkyOghZcDwHwOBWAAhihUNzr06xgyEq%2BQcbfJxm8badVdvgXAxJTGu1BKYfJPw1mhPZuE4RWyqmONqFKe%2BrjpGyLycwfgCCRDq2uSU2KZScb3NAZGG6yYpRQNcUoEQKgmQCDarADqlMv7FngLXBWBIP5-SWT0qKg11lEJqpUsBozpQUOhUQfiiADkCGgIQOwKo0UnlYdObq0AIIO1Sf8jRpStHlLsSM6p0oJGuKMPqV0zAAlIB5fghEuRdjrHaAIVwSyyj8IOEAA)

#### Code

```fsharp
Rollbar
  .RollbarLocator
  .RollbarInstance
  // .AsBlockingLogger(System.TimeSpan.FromSeconds 5)
  .Error(package, custom)
```
            

#### Result

```fsharp
Rollbar.RollbarLocator.RollbarInstance.Error(package, custom)

```
            
#### Problem description

The comment gets deleted.

Fantomas master branch at 1/1/1990

Default Fantomas configuration

**Comment by nojaf** [comment]
Hi Paul, thank you for reporting this issue.

The problem feels related to https://github.com/fsprojects/fantomas/issues/2027, although it isn't an exact duplicate.
The key problem is that we are currently using a rather deeply nested active pattern to get the `Rollbar.RollbarLocator.RollbarInstance.Error` from the AST. And that active pattern returns the long ident as a concatenated string without any regard to the individual ident ranges.

In short, though case to crack I'm afraid.

**Comment by nojaf** [comment]
Fix available in https://www.nuget.org/packages/fantomas-tool/4.6.2

**Comment by pbiggar** [comment]
> In short, though case to crack I'm afraid.

** 4 hours later ** 

> Fix available in https://www.nuget.org/packages/fantomas-tool/4.6.2

😂

Thanks!
